routers/common/middleware.go

```diff
@@ -81,3 +83,31 @@ func Middlewares() []func(http.Handler) http.Handler {
 	})
 	return handlers
 }
+
+func stripSlashesMiddleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
+		var urlPath string
+		rctx := chi.RouteContext(req.Context())
+		if rctx != nil && rctx.RoutePath != "" {
+			urlPath = rctx.RoutePath
+		} else {
+			urlPath = req.URL.Path
+		}
+
+		var sanitizedPath string
+		if len(urlPath) > 1 {
+			sanitizedPath = path.Clean(urlPath)
+
+			if string(urlPath[len(urlPath)-1]) == "/" {
+				sanitizedPath += "/"
+			}
```

Comment:
The previous code explicitly was removing a trailing slash. Why is this being added back in?

Reply:
I added this back in thinking this was causing the test case to fail. Just to test it out once.